src/lib/jest-globals.js

```diff
@@ -0,0 +1,74 @@
+function notImplemented() {
+	throw Error(`Not Implemented`);
+}
+
+// @todo expect.extend() et al
+
+global.jest = {
```

Comment:
Looks like those files here are not transpiled at all. Both `global` and `require` doesn't exist. Maybe we need to pass this file through webpack?

Reply:
ah - my bad, I meant to hang this off `window`. Probably better to inject it as a Webpack entry though, yeah.